### PR TITLE
WIP: Migration from ESP Journal

### PR DESCRIPTION
**Pull request by georgik**
Migrate articles from ESP Journal.

### CODE REVIEW

**Comment by georgik** [comment]
@pedrominatel @f-hollow Here's the initial test of porting one of articles.
@Hahihula - it was necessary to change following things manually:
- author changed from "Pedro Minatel" to "pedro-minatel"
- date format changed: 2021-02-13
- headers contains bold, which is not necessary:
```
# **Introduction
```
- title is duplicated with header, it can be removed from body: `# **How to use custom partition tables on ESP32`
- changed `showAuthor: false` - value true caues to display default author Espressif Developer Portal

**Review by f-hollow — COMMENTED** [review]
Improved readability of the initial paragraphs. The rest of the article can be improved after we migrate all other relevant articles.

**Another suggestion**: URL slugs shouldn't be this long. Suggest maybe boiling it down to `custom-partition-tables-esp32`.

File: content/blog/how-to-use-custom-partition-tables-on-esp32/index.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+title: "How to use custom partition tables on ESP32"
+date: 2021-02-13
+showAuthor: false
+authors:
+    - "pedro-minatel"
+---
+# How to use custom partition tables on ESP32
+
+
+# Introduction
```
Comment (f-hollow) [inline content/blog/how-to-use-custom-partition-tables-on-esp32/index.md:11]: Apparently, this heading and all the other ones below should be deeper by one level respectively.

```suggestion
## Introduction
```

**Comment by igrr** [comment]
Regarding the page URL, can we create an alias from `/how-to-use-custom-partition-tables-on-esp32-69c0f3fa89c8` (which is the [page URI](https://blog.espressif.com/how-to-use-custom-partition-tables-on-esp32-69c0f3fa89c8) on Medium) to `/blog/how-to-use-custom-partition-tables-on-esp32/`?

This will allow us to point blog.espressif.com to the new location later.

(https://gohugo.io/methods/page/aliases/)

**Comment by georgik** [comment]
Closing. This is handled by #105

**Pull request closed without merging**